src/qibolab/_core/native.py

```diff
@@ -78,8 +80,11 @@ def R(self, theta: float = np.pi, phi: float = 0.0) -> PulseSequence:
 
         ``theta`` will be the angle of the rotation, while ``phi`` the angle that the rotation axis forms with x axis.
         """
-        assert self.RX is not None
-        return rotation(self.RX, theta, phi)
+        if self.RX90 != None:
+            return rotation(self.RX, 2 * theta, phi)
```

Comment:
Shouldn't this be
```suggestion
        if self.RX90 is not None:
            return rotation(self.RX90, 2 * theta, phi)
```
?

Reply:
Yes, thank you for noticing!